index.html

```diff
@@ -42,5 +41,182 @@ <h5>LET'S CONNECT</h5>
+                    <img src="assets/images/portfolio_snapshoot/snapshoot3.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <div class="intro-text-works">Tonic</div>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <div class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <div class="works-flex-item">
+                    <div class="lang-container">
+                        <div class="lang">html</div>
+                        <div class="lang">css</div>
+                        <div class="lang">javascript</div>
+                    </div>
+                   
+                </div>
+                <div class="works-flex-item">
+                    <div class="btn-see">
+                        <a href="#" class="btn-see-project">See project</a>
+                    </div>
+                </div>
+            </div>
+            <div class="works-flex">
+                <div class="works-flex-item work-image">
+                    <img src="assets/images/portfolio_snapshoot/snapshoot4.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <div class="intro-text-works">Multi-Post <br/> Stories</div>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <div class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <div class="works-flex-item">
+                    <div class="lang-container">
+                        <div class="lang">html</div>
+                        <div class="lang">css</div>
+                        <div class="lang">javascript</div>
+                    </div>
+                   
+                </div>
+                <div class="works-flex-item">
+                    <div class="btn-see">
+                        <a href="#" class="btn-see-project">See project</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="aboutme">
+            <div class="aboutme-wrapper">
+                <div class="headline-container">
+                    <div class="intro-text-about">About<br> Myself</div>
```

Comment:
- [ ] As I suggested earlier it's important to keep an out for using correct semantic HTML. Please kindly use a heading tag here (e.g `h2` `h3` `h4` .etc).

something like this is all you need to do:
```suggestion
                    <h2 class="intro-text-about">About<br> Myself</h2>
```